apps/ejabberd/src/mod_muc_light_commands.erl

```diff
@@ -68,6 +69,22 @@ commands() ->
        ]},
       {result, {name, binary}}],
 
+     [{name, create_identifiable_muc_light_room},
```

Comment:
I've mentioned in other comment but: I think that creating unique room is a special case of creating room with defined username (not the other way round). This is how it's specified in XEP actually.

I think it would be cleaner to either have `create_room` and `create_unique_room` functions or even single `create_room` that will create unique group when `Id == <<>>`.